recipe/meta.yaml

```diff
@@ -9,30 +9,28 @@ source:
   sha256: c2e92d25d8bfd2ddff22ae47c4532c884678bac6ea0d935367eced051ae08bb3
 
 build:
-  number: 0
+  number: 1
   skip: True  # [win or py3k]
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
     - python
     - pip
-    - python-ecmwf_grib
   run:
     - python
-    - python-ecmwf_grib
+    - iris >=2
+    - python-eccodes
```

Comment:
Is there any way to specify "either package A or package B"? It will still work with python-ecmwf_grib.

Reply:
I don't think conda can do that. We could remove both and let the user install them manually, not sure if that is a good idea though.

Reply:
No, let's use python-eccodes.

Reply:
Another alternative would be to bump the build number and create another release with `python-ecmwf_grib` and let conda decided which one to pick based on what the users specifies.

Reply:
In that case it probably would be better to have the `python-eccodes` one having the higher build number. That way if the user doesn't specify either they will get eccodes, which is probably what we want.